tasks/messages/message_test.go

```diff
@@ -233,6 +238,34 @@ func TestParseMessageParams(t *testing.T) {
 			wantEncoded: "{\"Recoveries\":[{\"Deadline\":1,\"Partition\":1,\"Sectors\":{\"Count\":10,\"RLE\":[1,10]}}]}",
 			wantErr:     false,
 		},
+		// from https://github.com/filecoin-project/lily/issues/892
```

Comment:
without 691f5c2e39d850695a14c3cfd20d6a26cde9cf00 I have verified this test fails with:
```
message_test.go:292: got unexpected error: failed to parse message params method: 2, actor code: bafkqafdgnfwc6nrpobqxs3lfnz2gg2dbnzxgk3a, params: ��U�"��#t�`�":�d_z�Q  @�WF �v�� �Xa�oUp�ٮ}�2@��I��enw�GYWI�wӱ'�����<KM�$��s�;����J���[��G)<=��n��eIH$�Tׁ��[R,q�|d���,@: failed to override message param json marshaller: reflect: call of reflect.Value.NumField on zero Value
```